components/calendar/calendar.js

```diff
@@ -519,7 +518,7 @@ class Calendar extends LocalizeCoreElement(RtlMixin(LitElement)) {
 								icon="tier1:chevron-right">
 							</d2l-button-icon>
 						</div>
-						<table aria-labelledby="${labelId}" role="presentation">
+						<table aria-labelledby="${labelId}">
```

Comment:
Is the role="gridcell" still needed on the `td` if we're removing the `role="presentation"`? On Mac+Chrome at least, that's the default role when native table semantics are in play.

Reply:
I don't think it _should_ be needed. But remove it, and Firefox doesn't know how to read the elements 🤦. (At least, that's what I think is happening)

Reply:
Just a quick follow-up note for anyone reading this later. Chrome and Safari default `td` to `gridcell` role, whereas Firefox defaults it to `cell`. As per `aria-selected` specs, the role must be `gridcell`, `option`, `row`, or `tab`.